src/app/components/privacy-policy/privacy-policy.component.scss

```diff
@@ -50,10 +50,11 @@
     }
     .scroll-to-top {
       position: fixed;
-      top: 50%;
-      right: 4%;
+      bottom: 2%;
```

Comment:
Why was this change made?

Reply:
It positions the scroll top button at the bottom right of the privacy page

Reply:
Maybe the earlier one is better! You could revert this change.